flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/LogicalFunctionsITCase.java

```diff
@@ -35,10 +35,104 @@
     @Parameterized.Parameters(name = "{index}: {0}")
     public static List<TestSpec> testData() {
         List<TestSpec> cases = new ArrayList<>();
+        cases.addAll(and());
```

Comment:
side comment: I just wanted to ask for dropping tests that use the old test bases but I couldn't find any tests for it, weird

Reply:
Yeah, I noticed there were none, which is why I had added these.